config/common/errors.go

```diff
@@ -47,6 +47,9 @@ var (
 	ErrUnknownBootDeviceLayout = errors.New("layout must be one of: aarch64, ppc64le, x86_64")
 	ErrTooFewMirrorDevices     = errors.New("mirroring requires at least two devices")
 
+	// partition
+	ErrWrongPartitionNumberForRoot = errors.New("new partition using reserved label \"root\"")
```

Comment:
This is nice and concise, but I think it'll really confuse users who didn't know they needed to set `number`.